packages/plugin-dom-editable/test/eventNormalizerUtils.ts

```diff
@@ -1,5 +1,4 @@
 import { EventBatch, EventNormalizer } from '../src/EventNormalizer';
-import { nodeName } from '../../utils/src/utils';
```

Comment:
Please change the commit message to one of the following:
`[FIX] DomEditable: remove unused import`
`[IMP] DomEditable: remove unused import`
`[REM] DomEditable: remove unused import`

Reply:
done